tests/test_folium.py

```diff
@@ -470,8 +470,8 @@ def test_topo_json(self):
         assert ''.join(topojson_str.split())[:-1] in ''.join(out.split())
 
         bounds = self.map.get_bounds()
-        assert bounds == [[-124.56617536999985, 41.99187135900012],
-                          [-116.46422312599977, 46.28768217800006]], bounds
+        assert bounds == [[41.99187135900012, -124.56617536999985,],
+                          [46.28768217800006, -116.46422312599977]], bounds
```

Comment:
To pass the tests, you just need to remove extra comma after `-124.56617536999985`.